### ISSUE TITLE
Issue about detection result...

### ISSUE THREAD

**Issue by Gogotis** [LeeDongYeun/keras-m2det#2]
Hi, Thank you for sharing your Project about M2det.

Training config 
:  COCO dataset (2017), VGG16 backbone network, Input size 320x320, ...
: tensorflow-gpu==1.14.0

So, running code like this

`python3 keras_m2det/bin/train.py --backbone vgg16 --image-min-side 320 --image-max-side 320 coco /<coco-dataset-path>/COCO/`

All loss(loss, regression, classification) is almost 1.7~2.1(training during 3days). 

As a result,  drawing bounding boxes of small object has good performance, but big object not. 
Also, prediction scores are very low...

![000000184613](https://user-images.githubusercontent.com/60951742/74311488-f77e7a00-4db2-11ea-94ad-22e4702a907f.jpg)

![result](https://user-images.githubusercontent.com/60951742/74311533-0e24d100-4db3-11ea-9d4a-d1ec904889a6.png)

** I used the training dataset image to check accuracy...

Do you have any idea to solve this problem...?
